e2e/jenkins/README.md

```diff
@@ -8,50 +8,72 @@
-- This file contains the password for user *admin*.
-- Once logged in it redirects to a *Getting Started* page. Click on *Install Suggested Plugins* option.
+
+   ```
+   http://<jenkins-server>:8080
+   ```
+
+- The first time this URL is opened, the user is provided with a path to a file in the server _/var/lib/jenkins/secrets/initialAdminPassword_.
+- This file contains the password for user _admin_.
+- Once logged in it redirects to a _Getting Started_ page. Click on _Install Suggested Plugins_ option.
 - The user is now prompted to create a new admin user.
-- Create a user called *openebs* providing the essential details requested by the *Create First Admin User* page and click on *Save and Finish*.
-- You should be seeing a message which says *Jenkins is ready!*
-- Click on *Start using Jenkins*
-- You will logged in as user *openebs* automatically and will be able to see his *Dashboard*.
+- Create a user called _openebs_ providing the essential details requested by the _Create First Admin User_ page and click on _Save and Finish_.
+- You should be seeing a message which says _Jenkins is ready!_
+- Click on _Start using Jenkins_.
+- You will logged in as user _openebs_ automatically and will be able to see his _Dashboard_.
 
-Note:
-```
-By default the pages in Jenkins do not refresh automatically, to enable it click on the *ENABLE AUTO REFRESH* link on the top right of the page. 
-```
+__Note:__
+
+   ```
+   By default the pages in Jenkins do not refresh automatically, to enable it click on the *ENABLE AUTO REFRESH* link on the top right of the page.
+   ```
 
 ## Continuous Integration - Step 3: Installing Additional Plugins
 
-- On the left side of the *Dashboard* you have links for managing Jenkins.
-- Click on *Manage Jenkins->Manage Plugins*, you are now in the Plugin Manager page.
-- You should be able to see a page with 4 tabs(*Updates-Available-Installed-Advanced*).
-- Click on the *Available* tab and in the *Filter* textbox search and add following 2 plugins one by one.
+- On the left side of the _Dashboard_ you have links for managing Jenkins.
+- Click on _Manage Jenkins->Manage Plugins_, you are now in the _Plugin Manager_ page.
+- You should be able to see a page with 4 tabs(_Updates-Available-Installed-Advanced_).
+- Click on the _Available_ tab and in the _Filter_ textbox search and add following 3 plugins one by one.
   - Environment Injector Plugin
   - Hudson Post build task
   - Slack Notification Plugin
-- Select the *Restart Jenkins when installation is complete and no jobs are running* to restart jenkins.
+- Select the _Restart Jenkins when installation is complete and no jobs are running_ to restart jenkins.
+
+### Configure Slack Plugin Preferences
 
+__Note:__
 
-## Continuous Integration - Step 6: Creating The Jobs
+```
+It is a prerequisite that the Jenkins app for Slack is already installed and configured.
```

Comment:
Prerequisite
========
Jenkins application for Slack must be installed and configured.